docs/product-getting-started-guide/src/main/asciidoc/test-violation-based-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='testing_violation_numbers-proc']
+= Testing the multiple violations scenario
+
+The following chapter instructs you how to copy the *Suspend due to total points* asset and modify it to create the driver license suspension scenario for drivers with multiple violations using the data that you specified when you set the Driver License Suspension rules and actions.
+
+[float]
+== Prerequisites
+
+* Created the Driver department traffic violations project
```

Comment:
Again monospaced formatting, etc. See my previous comments on the last few prereqs.

Reply:
done